src/Jackalope/NodeType/NodeProcessor.php

```diff
@@ -262,12 +279,11 @@ private function assertValidProperty($property)
                     $values = array($values);
                 }
                 foreach ($values as $value) {
-                    if (!preg_match(self::VALIDATE_PATH, $value)) {
-                        throw new ValueFormatException(sprintf(
-                            'Value "%s" for PATH property at "%s" is invalid. Segments are separated by / and allowed chars are -a-zA-Z0-9:_',
-                            $value,
-                            $property->getPath()
-                        ));
+                    try {
+                        // making the path absolute also validates the result to be a valid path
+                        PathHelper::absolutizePath($value, $property->getNode()->getPath());
```

Comment:
what do you think of this solution? this will be very permissive, but consistent with other path validations.